tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -2625,6 +2668,13 @@ namespace Tensile
             }
         }
 
+        // workspace for amaxD
+        if(problemType.outputAmaxD)
+        {
+            auto numWGS = getNumWorkGroups(problem, sizeMapping);
+            size += problem.amaxd().elementBytes() * numWGS;
+        }
+
         // Custom kernel synchronizer
```

Comment:
As per review suggestion, WS only requires sizeof(element) * the-num-work-groups, don't use hardcoded 4096